### DISCUSSION TITLE
Duplicate User Group

### DISCUSSION THREAD

**Discussion by wbdzynr** [craftcms/cms#15352] [Ideas]
When creating a user group, it would be nice we had a "Duplicate" option like other parts of the CMS.  My current project requires the admin to create new user groups for each of their committees.  I'd like to be able to set up one with all the proper permissions, then allow them to duplicate it moving forward.

**Comment by JayBox325** [comment]
This is something I need too, I think this plugin could be an option: https://plugins.craftcms.com/cloner

**Reply by wbdzynr** [reply]
Thanks for the suggestion!

**Reply by JayBox325** [reply]
I've been using it since I found it and it works great.
